CHANGES

```diff
@@ -1,6 +1,10 @@
 django-simple-menu changelog
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+Version 2.0.0 - Released ...
+- Dropped support for Django < 2.2
+- Dropped support for Python < 3.6
+
```

Comment:
Obviously this is quite the step, but the old Python and Django versions aren't really supported anymore and we should use the oppurtunity to make the jump.